src/Checkbox.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Checkbox(Props) {
+	return <input type="checkbox" name={Props.row} data-key={Props.index} value={Props.value} checked={Props.checked} style={{flex:1}} onChange={Props.onChangeHandler} disabled={Props.disabled}/>
+}

```

Comment:
```suggestion
const Checkbox = (props) => ( 
<input type="checkbox" name={props.row} data-key={props.index} value={props.value} checked={props.checked} style={{flex:1}} onChange={props.onChangeHandler} disabled={props.disabled}/>
)

```

Another very useful pattern is to destructure the props beforehand, this way you can get rid of the repetitive `props.` notation:

```suggestion
const Checkbox = ({row, index, value, checked, onChangeHandler, disabled}) => (
<input type="checkbox" name={row} data-key={index} value={value} checked={checked} style={{flex:1}} onChange={onChangeHandler} disabled={disabled}/>
)

```

Reply:
Note that in this case you can do an implicit return using parentheses, so you don't have to add `return` again.

However, if you have a component where you need to do some other stuff before returning (like maintaining state), you need curly braces and an explicit return.